trashcli/list_mount_points.py

```diff
@@ -49,26 +49,23 @@ class mntent_struct(Structure):
     if sys.platform == "cygwin":
         libc_name = "cygwin1.dll"
     else:
-        libc_name = find_library("c")
-
-    if libc_name == None :
-        libc_name="/lib/libc.so.6" # fix for my Gentoo 4.0
-
+        libc_name = util.find_library("c") or "/lib/libc.so.6" # fix for my Gentoo 4.0
     libc = cdll.LoadLibrary(libc_name)
     libc.getmntent.restype = POINTER(mntent_struct)
-    libc.fopen.restype = c_void_p
+    PyFile_AsFile = pythonapi.PyFile_AsFile
+    PyFile_AsFile.argtypes = [pythonapi.py_object]
 
-    f = libc.fopen("/proc/mounts", "r")
-    if f==None:
-        f = libc.fopen("/etc/mtab", "r")
-        if f == None:
+    try:
```

Comment:
I used the libc.fopen because is the documented way to use getmntent on linux.
Why do you changed that?